src/frontend/sider/Fagsak/Behandling/BehandlingRouter.tsx

```diff
@@ -90,7 +90,7 @@ const BehandlingRouter: React.FC<Props> = ({ bruker, fagsak }) => {
                     <SimuleringProvider åpenBehandling={behandling}>
                         <VedtakStegProvider åpenBehandling={behandling}>
                             <SammensattKontrollsakProvider åpenBehandling={behandling}>
-                                <OppsummeringVedtak åpenBehandling={behandling} bruker={bruker} />
+                                <Vedtak åpenBehandling={behandling} bruker={bruker} />
```

Comment:
Nå heter contexten `VedtakSteg` og komponenten `Vedtak`. Burde disse kanskje vært samme? Kanskje fjerne "steg" fra provider?

Reply:
Tenker du da på å kalle de forskjellige delene i konteksten for `VedtakProvider`, `VedtakContext`, `useVedtakContext`? Og at komponenten heter `Vedtak`? Det kan gi mening ja hvis de er tett koblet 😊

Reply:
Jepp var det jeg tenkte! Contexten wrappes rundt hele vedtak-siden, og de andre sidene har jo ikke "steg" i navnet på tilhørende context, så tenker det gir mening 😊

Reply:
Enig!